src/main/java/org/broadinstitute/consent/http/db/DatasetDAO.java

```diff
@@ -188,9 +189,12 @@ Integer insertDataset(
                 " SET name = :datasetName," +
                     " update_date = :updateDate, " +
                     " update_user_id = :updateUserId, " +
-                    " needs_approval = :needsApproval " +
+                    " needs_approval = :needsApproval, " +
+                    " dac_id = :dacId " +
                 " WHERE dataset_id = :datasetId")
```

Comment:
Not a request, but something to consider - I think sql in text blocks is much easier to read. We should move to that format as needed.

Reply:
[DatasetServiceDAO](https://github.com/DataBiosphere/consent/blob/744c6b2838875d82f4a9fd8982ef48ac887d1549/src/main/java/org/broadinstitute/consent/http/service/dao/DatasetServiceDAO.java)